.pipelines/nuget.yml

```diff
@@ -1,107 +1,33 @@
+parameters:
+- name: DoCompliance
+  displayName: Run Compliance Tasks?
+  type: boolean
+  default: true
+
+- name: DoEsrp
+  displayName: Run code sign tasks? Must be true if you are doing an OnnxRuntime extensions release.
+  type: boolean
+  default: true
+
+- name: IsReleaseBuild
+  displayName: Is this a release build? Set it to true if you are doing an OnnxRuntime extensions release.
+  type: boolean
+  default: false
+
+trigger:
+  branches:
+    exclude:
+      - "*"
+
+pr:
+  branches:
+    include:
+      - "*"
```

Comment:
maybe running this for every PR is too frequent?

also I think for other pipelines we just specify the triggers in the web UI instead of the yml file. could do that for this as well to be consistent.

Reply:
Yep